doc/build/kconfig/tips.rst

```diff
@@ -748,7 +748,7 @@ defaults to n* instead of *n if <dependencies, possibly inherited>*).
 .. note::
 
    The one case where ``default n``/``default ""`` is not redundant is when
-   defining a symbol in multiple locations and wanting to override e.g. a
+   defining a symbol in multiple locations and wanting to override e.g. to a
    ``default y`` on a later definition.
```

Comment:
This is not what the sentence was trying to say.

What the original sentence was trying to tell the reader is that normally:
```
config MY_SETTING
    bool
    default n
```
is redundant, except for the situation where `MY_SETTING` is defined in multiple locations, and you want to overwrite the `default y` that comes on a later definition.

For example the following cases:

1) 
```
config MY_SETTING
    bool
    default n
```
is identical to this:
```
config MY_SETTING
    bool
```
which means the `default n` in the first writing is redundant, and the `default n` is not needed here.

2)
```
config MY_SETTING
    bool

... Other, later sourced Kconfig file

config MY_SETTING
    bool
    default y
```
in this case, the default value for `MY_SETTING` becomes `y`.

3)
```
config MY_SETTING
    default n
    bool

... Other, later source file
config MY_SETTING
    bool
    default y
```

in this case, the first `default n` wins, and because a later definition of the same setting has `default y`, then the `default n` is no longer redundant.

This is useful, for example in samples or Zephyr modules where you want to ensure `n` is default in cases where Zephyr might have defined the default to `y`.

This is the intention of the existing text, which is not what this PR now describes.